temoa/components/geography.py

```
@@ -77,7 +77,9 @@ def RegionalGlobalInitializedIndices(M: 'TemoaModel'):
 # ============================================================================
 
 
-def RegionalExchangeCapacity_Constraint(M: 'TemoaModel', r_e, r_i, p, t, v):
+def RegionalExchangeCapacity_Constraint(
+    M: 'TemoaModel', r_e: str, r_i: str, p: int, t: str, v: int
+) -> Any:
```

Ruff (0.14.0)

Dynamically typed expressions (typing.Any) are disallowed in `RegionalExchangeCapacity_Constraint`

(ANN401)